README.md

```diff
@@ -45,7 +45,22 @@ For more information about inheriting configuration from a gem please check
 [RuboCop's
 documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-configuration-from-a-dependency-gem).
 
-## General
+## Table of Contents
+
+* [General](#general)
+* [Formatting](#formatting)
+* [Syntax](#syntax)
+* [Naming](#naming)
+* [Classes and Modules](#classes-and-modules)
+* [Exceptions](#exceptions)
+* [Collections](#collections)
+* [Strings](#strings)
+* [Regular Expressions](#regular-expressions)
+* [Percent Literals](#percent-literals)
+* [Testing](#testing)
+* [The Rest](#the-rest)
+
+## General <a name="general"></a>
```

Comment:
No need to add anchors, markdown headers have them created on the site and in github automatically:
 - https://shopify.github.io/ruby-style-guide/#general
 - https://github.com/Shopify/ruby-style-guide#general

Reply:
TIL! Thanks